SteamKit2/SteamKit2/Util/Utils.cs

```diff
@@ -168,49 +150,13 @@ public static EOSType GetOSType()
                         }
                     }
 
-                // Not currently used by Mono. Maybe .NET Core will use this someday?
-                case PlatformID.MacOSX:
-                    return EOSType.MacOSUnknown;
-
                 default:
                     return EOSType.Unknown;
             }
-#else
-#error Unknown Target Platform
-#endif
-        }
-
-        public static bool IsRunningOnDarwin()
-        {
-            // Replace with a safer way if one exists in the future, such as if
-            // Mono actually decides to use PlatformID.MacOSX
-            var buffer = IntPtr.Zero;
-            try
-            {
-                buffer = Marshal.AllocHGlobal( 8192 );
-                if ( uname( buffer ) == 0 )
-                {
-                    var kernelName = Marshal.PtrToStringAnsi( buffer );
-                    if ( kernelName == "Darwin" )
-                    {
-                        return true;
-                    }
-                }
-            }
-            catch
-            {
-            }
-            finally
-            {
-                if ( buffer != IntPtr.Zero )
-                    Marshal.FreeHGlobal( buffer );
-            }
-
-            return false;
         }
 
-        [DllImport ("libc")]
-        static extern int uname (IntPtr buf);
+        public static bool IsMacOS()

```

Comment:
This doesn't appear to be needed outside of this file, so maybe get rid of this function completely?